### PR TITLE
Added title to each page.

### LINKED ISSUE #96: Add title to each page
Currently each page's title is its URL, add a title for every page

### PR DESCRIPTION
**Pull request by labeeb-7z**
resolves #96

### CODE REVIEW

**Review by kkothari2001 — CHANGES_REQUESTED** [review]
I guess it would be better to  add the Head components to all files in the `pages/` directory instead of all individual components (some of which I can see have been missed out)

**Comment by kkothari2001** [comment]
Please merge with main to get latest changes and then add title to sponsors page too

File: components/Projects/ProjectSelection/ProjectSelection.js [hunk 1]
```diff
@@ -3,20 +3,22 @@ import Link from 'next/link';
 
 const ProjectSelection = () => {
   return (
-    <div className={styles.projSelect}>
```
Comment (kkothari2001) [inline components/Projects/ProjectSelection/ProjectSelection.js:6]: I'm not sure if I understand why some components have to be put in fragments. What motivates this?

Comment (labeeb-7z) [reply components/Projects/ProjectSelection/ProjectSelection.js:6]: I added fragments when I included the Head component in individual components.
when i moved head to the main page of each route, i missed to remove the fragments from some components.
Just removinig them now.

Comment (labeeb-7z) [reply components/Projects/ProjectSelection/ProjectSelection.js:6]: I noticed some unnecessary fragments in the following files, which were not modified by me earlier
image
image

OurReach.js
Footer.js
OurSponsors.js
HomeNotifications.js
Navbar.js

should I remove those as well? or only the ones I modified.

Comment (kkothari2001) [reply components/Projects/ProjectSelection/ProjectSelection.js:6]: The codebase does need cleanup, but let's do that seperately. We shouldn't be mixing concerns at the moment.

**Review by kkothari2001 — APPROVED** [review]

**Pull request merged**